src/cli.ts

```diff
@@ -33,7 +33,7 @@ program
   .option('--ct', 'install Playwright Component testing')
   .option('--quiet', 'do not ask for interactive input prompts')
   .option('--gha', 'install GitHub Actions')
-  .option('--lang <language>', 'language to use (js, TypeScript)', 'TypeScript')
```

Comment:
we actually wanted to drop js 😄

Reply:
oh, then let's remove all the code that goes with it!